modules/library/jdbc/src/test/java/org/geotools/jdbc/JDBCGroupByVisitorOnlineTest.java

```diff
@@ -344,6 +345,30 @@ public void testSimpleGroupByWithSum() throws Exception {
         checkValueContains(value, "SCHOOL", "180.0");
     }
 
+    protected void testGroupByWithAggregateAllValuesNull(
+            Aggregate aggregate, boolean expectOptimized) throws Exception {
+        PropertyName aggregateAttribute =
+                CommonFactoryFinder.getFilterFactory().property(aname("fuel_consumption"));
+        Expression groupBy = dataStore.getFilterFactory().property(aname("building_type"));
+        List<Object[]> value =
+                genericGroupByTestTest(
+                        Query.ALL, aggregate, aggregateAttribute, expectOptimized, groupBy);
+        // All values in the fuel_consumption column are null, this test should return null without
+        // an NPE
```

Comment:
Nitpick to avoid line wrapping

```suggestion
        // All values in the fuel_consumption column are null, should return null without an NPE
```

Reply:
fixed